src/app/components/CommentsList/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { Text, View, TouchableOpacity } from 'react-native';
+import { Book } from '@interfaces/book';
+
+import { DATA } from './constants';
+import styles from './styles';
+import CommentCell from './Comment';
+
+interface Props {
+  book: Book;
+}
+
+// will need the book when I fetch the comments from the repository
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+function CommentList({ book }: Props) {
+  const data = {
+    data: DATA.slice(0, 2),
+    viewingAllComments: false
+  };
+  const [state, setState] = useState(data);
```

Comment:
Use 2 different states for this, one for `viewingAllComments` and another for `data`
```
const [commentList, setCommentList] = useState(DATA.slice(0, 2));
const [viewingAllComments, setViewingAllComments] = useState(false);
```